internal/storage/index/index.go

```diff
@@ -436,7 +436,7 @@ func (idx *index) RepoStats(_ context.Context) storage.RepoStats {
 }
 
 func (idx *index) Reload(ctx context.Context) ([]storage.Event, error) {
-	log := ctxzap.Extract(ctx).Sugar()
+	log := zap.L().Named("index").Sugar()
```

Comment:
Why not use the logger from the context?

I think I would have passed the store logger in the context to this function and added the start/end log call around it because this is where the meat of the logic is.